cmake/SndFileConfig.cmake.in

```diff
@@ -5,19 +5,24 @@ set(SndFile_VERSION_PATCH @PROJECT_VERSION_PATCH@)
 
 set (SndFile_WITH_EXTERNAL_LIBS @SndFile_WITH_EXTERNAL_LIBS@)
 set (SndFile_WITH_MPEG @SndFile_WITH_MPEG@)
+set (SndFile_STATIC NOT @BUILD_SHARED_LIBS@)
 
 @PACKAGE_INIT@
 
 include (CMakeFindDependencyMacro)
 
-if (SndFile_WITH_EXTERNAL_LIBS AND NOT @BUILD_SHARED_LIBS@)
+if ((SndFile_WITH_EXTERNAL_LIBS OR SndFile_WITH_MPEG) AND SndFile_STATIC)
+    list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+endif()
+
+if (SndFile_WITH_EXTERNAL_LIBS AND SndFile_STATIC)
 	find_dependency (Ogg 1.3)
 	find_dependency (Vorbis)
 	find_dependency (FLAC)
 	find_dependency (Opus)
 endif ()
 
-if (SndFile_WITH_MPEG AND NOT @BUILD_SHARED_LIBS@)
```

Comment:
Why not just rename MPG123 to Mpg123 here?

Reply:
Because a quick search of both names threw the following results on the libsndfile directory:

~~~
$ grep --exclude-dir=.git -r Mpg123 | wc -l
4
$ grep --exclude-dir=.git -r MPG123 | wc -l
77
~~~

It was much less work to change only 4 lines and rename one file than the other option.

~~~
$ grep --exclude-dir=.git -r Mpg123
cmake/FindMpg123.cmake:#  MPG123_FOUND        - True if Mpg123 found.
cmake/FindMpg123.cmake:find_package_handle_standard_args (Mpg123
cmake/SndFileChecks.cmake:find_package (Mpg123 1.25.10)
Makefile.am:    cmake/Findmp3lame.cmake cmake/FindMpg123.cmake \
~~~